README.md

```diff
@@ -355,23 +355,25 @@ return GetMaterialApp(
 
 ## Change Theme
 
-Please do not use any higher level widget than GetMaterialApp in order to update it. This can trigger duplicate keys. A lot of people are used to the prehistoric approach of creating a "ThemeProvider" widget just to change the theme of your app, and this is definitely NOT necessary with Get.
+Please do not use any higher level widget than `GetMaterialApp` in order to update it. This can trigger duplicate keys. A lot of people are used to the prehistoric approach of creating a "ThemeProvider" widget just to change the theme of your app, and this is definitely NOT necessary with **GetX™**.
```

Comment:
prehistoric? aren't this the usual way of doing it without GetX?

Reply:
@jonataslaw

Reply:
Just for my understanding, `Get` needs to be replaced with `GetX™` everywhere?
Because there are still some places in this PR where it is `Get`

Reply:
@stefandevo the ideal is to replace everywhere